### PR TITLE
Add spell checker as a recommended extension and create vocabularly of repo specific exceptions.

### PR DESCRIPTION
**Pull request by polina-c**
The spell checker verifies comments and code: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

### CODE REVIEW

**Comment by polina-c** [comment]
@goderbauer 

Does it sound as a good idea?
If yes, what would be the best way to discuss with team? Discord hackers?

File: .vscode/settings.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "files.trimTrailingWhitespace": true,
     "cSpell.words": [
         "binpb",
+        "canonicalize",
```
Comment (goderbauer) [inline .vscode/settings.json:19]: What does this setting do? Can you link to its docs? I couldn't find them.

**Comment by goderbauer** [comment]
> Does it sound as a good idea?

I don't know. I don't know what this setting does (I couldn't find any docs for it). And I also don't use VSCode.

> If yes, what would be the best way to discuss with team? Discord hackers?

Socializing this on Discord to get feedback (from people that actually develop with VSCode) sounds like a good idea.

**Comment by polina-c** [comment]
https://discord.com/channels/608014603317936148/608021234516754444/1197335900594647141

**Pull request closed without merging**